src/main/java/com/yahoo/bullet/parsing/Projection.java

```diff
@@ -9,27 +9,27 @@
 import com.yahoo.bullet.common.BulletError;
 import com.yahoo.bullet.common.Configurable;
 import com.yahoo.bullet.common.Initializable;
+import com.yahoo.bullet.parsing.expressions.LazyExpression;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 
 @Slf4j @Getter @Setter
 public class Projection implements Configurable, Initializable {
-    /**
-     * The map of source field names to their new projected names.
-     */
     @Expose
-    private Map<String, String> fields;
+    private String name;
+
+    @Expose
+    private LazyExpression value;
```

Comment:
This can't be any LazyExpression right? Or are we allowing logical ands and stuff to appear here now?

Reply:
I'm not sure why we would want to restrict logical operations. We can have boolean fields and allowing logical operations wouldn't break anything.